app/views/pregnancies/_call_log.html.erb

```diff
@@ -3,14 +3,29 @@
 		<h1 class="border-bottom title"> Call Log</h1>
 		<table class="table border-side">
 		<tr>
-			<th>Time Stamp</th>
+			<th>Date</th>
+			<th>Time</th>
 			<th>Result</th>
 			<th>user</th>
 		</tr>
+		<div class="btn btn-primary" id="toggle-call-log">View all calls</div>
```

Comment:
As an enhancement, let's make it so this button only shows up if there are 10+ calls (e.g. if there are more calls to show, and toggle on and off). I think this might be a good easy PR for someone to get comfy with the codebase so let's leave it hanging for now.